activemq/publisherAMQP.go

```diff
@@ -0,0 +1,74 @@
+package activemq
+
+import (
+	"context"
+	"github.com/andile-innovation/popcorn/log"
+	"net"
+	"pack.ag/amqp"
+	"time"
+)
+
+type NewAMQPPublisherRequest struct {
+	Host string
+	Port int
+}
+
+func NewAMQPPublisher(request NewAMQPPublisherRequest) AMQPPublisher {
+	return AMQPPublisher{
+
+	}
+}
+
+type AMQPPublisher struct {
+	Client *amqp.Client
```

Comment:
add host and port, and credentials username and password (but make all private):
```suggestion
	client *amqp.Client
	conn net.Conn
	host string
	Port int
	username string
	password string
```